src/sagemaker_xgboost_container/algorithm_mode/serve_utils.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import csv
+import io
+import json
+import numpy as np
+import os
+
+from sagemaker_containers.record_pb2 import Record
+from sagemaker_containers._recordio import _write_recordio
+
+from sagemaker_xgboost_container.algorithm_mode import integration
+
+
+logging = integration.setup_main_logger(__name__)
+
+SAGEMAKER_INFERENCE_OUTPUT = 'SAGEMAKER_INFERENCE_OUTPUT'
+
+# xgboost objective learning tasks
+# https://xgboost.readthedocs.io/en/release_0.90/parameter.html#learning-task-parameters
+REG_SQUAREDERR = 'reg:squarederror'
```

Comment:
Let's move these constants, since these are the same values regardless of algorithm mode or script mode. There is a directory for constants already.